src/java/bobbybot/ToDo.java

```diff
@@ -0,0 +1,30 @@
+package bobbybot;
+
+/** Represents a task without any time restrictions
+ *  @author mokdarren
+ */
+public class ToDo extends Task{
```

Comment:
Since "todo" is one word, perhaps naming it `Todo` would be more appropriate than `ToDo`.